.github/autoissuelabeler.yml

```diff
@@ -51,12 +51,41 @@
   title: "(?i).*mongo.*"
   description: "(?i).*mongo.*"
   notify: [loicmathieu]
-- labels: [area/openapi]
+- labels: [area/openapi, area/smallrye]
   title: "(?i).*openapi.*"
+  description: "(?i).*openapi.*"
   notify: [EricWittmann,MikeEdgar,phillip-kruger]
-- labels: [area/graphql]
+- labels: [area/graphql, area/smallrye]
   title: "(?i).*graphql.*"
-  notify: [phillip-kruger,jmartisk]  
+  description: "(?i).*graphql.*"
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/tracing, area/smallrye]
+  title: "(?i).*(trace|tracing|telemetry).*"
+  description: "(?i).*(trace|tracing|telemetry).*"
+  notify: [kenfinnigan,Ladicek]
+- labels: [area/config, area/smallrye]
+  title: "(?i).*config.*"
```

Comment:
OK so I maintain we need that one removed because it will match all bugs using the default template.

Reply:
removed.